### DISCUSSION TITLE
81

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#54] [General]
# 潮流周刊第81期 - 良渚的🦌

照片拍于周五下午和后端同学去良渚徒步团建，良渚公园着实很大很草原，其中有一个鹿苑，挺可爱的。

https://weekly.tw93.fun/posts/81-%E8%89%AF%E6%B8%9A%E7%9A%84%F0%9F%A6%8C

**Comment by kevinwang0224** [comment]
xterm.js 和 fx 的图贴错啦
